templates/alertmanager.yaml

```
@@ -44,3 +49,4 @@ spec:
   secrets:
     - alertmanager-k8s-tls
     - alertmanager-k8s-proxy
+{{ .Affinity }}
```

```suggestion
{{ .Affinity | indent 2 }}
```

This way the indentation level depends on the resource itself